stage0/entrypoint.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 const (
-	enterEntrypoint = "coreos.com/rkt/stage1/enter"
-	runEntrypoint   = "coreos.com/rkt/stage1/run"
-	gcEntrypoint    = "coreos.com/rkt/stage1/gc"
+	enterEntrypoint = "stage1-enter"
+	runEntrypoint   = "stage1-run"
+	gcEntrypoint    = "stage1-gc"
```

Comment:
Sorry. I started changing this on the weekend but internet dropped out and I never put up the PR. Stand by...